### PR TITLE
[WIP] More tests

### PR DESCRIPTION
**Pull request by VicDeo**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
nice 4% more coverage - https://scrutinizer-ci.com/g/owncloud/files_antivirus/inspections/ae47e55d-2c46-4663-a264-9664290531b7

**Comment by DeepDiver1975** [comment]
@VicDeo let's merge this and continue in a second PR - what do you think? THX

File: tests/scannertest.php [hunk 1]
```diff
@@ -54,16 +54,6 @@ public function setUp() {
 		);
 	}
 	
-	public function testBackgroundScan(){
```
Comment (DeepDiver1975) [inline tests/scannertest.php]: where did this test go? no longer necessary?

Comment (VicDeo) [inline tests/scannertest.php]: @DeepDiver1975 to cron/tasktest.php

**Comment by VicDeo** [comment]
@DeepDiver1975 Code quality increased by 0.01 as a bonus.
Merging if you have no objections.

**Comment by DeepDiver1975** [comment]
> Code quality increased by 0.01 as a bonus.

:tada: Great! :wink:

**Pull request merged**